docs/framework/network-programming/automatic-proxy-detection.md

```diff
@@ -38,7 +38,7 @@ Automatic proxy detection is a process by which a Web proxy server is identified
 > [!NOTE]
 > Applications running as an NT Service or as part of ASP.NET use the Internet Explorer proxy server settings (if available) of the invoking user. These settings may not be available for all service applications.  
   
- Proxies are configured on a per-connectoid basis. A connectoid is an item in the network connection dialog, and can be a physical network device (a modem or Ethernet card) or a virtual interface (such as a VPN connection running over a network device). When a connectoid changes (for example, a wireless connection changes an access point, or a VPN is enabled), the proxy detection algorithm is run again.  
+ Proxies are configured on a per-connection basis. A connection is an item in the network connection dialog, and can be a physical network device (a modem or Ethernet card) or a virtual interface (such as a VPN connection running over a network device). When a connection changes (for example, a wireless connection changes an access point, or a VPN is enabled), the proxy detection algorithm is run again.  
```

Comment:
Not sure this one should have changed?

Reply:
Maybe not, the term didn't appear to be used anywhere else, but maybe it is an older Windows term https://www.pcmag.com/index.php/encyclopedia/term/connectoid